src/js/components/Menu/doc.js

```diff
@@ -103,10 +103,10 @@ The object values can be any Button prop, for example: label and onClick.`,
 };
 
 export const themeDoc = {
-  'global.colors.control': {
+  'menu.icons.color': {
```

Comment:
I'm thinking we should still keep 'global.colors.control' with the actual base.js default value. to be more explicit.

Reply:
> Looks good, one tiny comment.
> we might want to solve the test issues on a separate PR.

I believe this PR is aiming to solve the test issues: https://github.com/grommet/grommet/pull/4030